pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+	<!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+	integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
+	<title>{{title}}</title>
+</head>
+<body>
+
+	{% if messages %}
+		{% for message in messages %}
+			<div class="alert alert-{{ message.tags }}">
+				{{ message }}
+			</div>
+		{% endfor %}
+	{% endif %}
+	
+	<div class = "container">
```

Comment:
you can add an id = "content" to the div tag, might be useful for later and I think it's a good practice also appeared in the basic django tutorial - **(https://docs.djangoproject.com/en/3.1/ref/templates/language/)**

Reply:
instead of <div class = "container">  you can change it to <div id = ''content" class = "container">

Reply:
I made the change